internal/view/cronjob.go

```diff
@@ -64,9 +67,106 @@ func jobCtx(path, uid string) ContextFunc {
 func (c *CronJob) bindKeys(aa ui.KeyActions) {
 	aa.Add(ui.KeyActions{
 		tcell.KeyCtrlT: ui.NewKeyAction("Trigger", c.trigger, true),
+		ui.KeyShiftS:   ui.NewKeyAction("Suspend", c.setSuspend, true),
+		ui.KeyShiftU:   ui.NewKeyAction("Unsuspend", c.setUnsuspend, true),
 	})
 }
 
+func (c *CronJob) setSuspend(evt *tcell.EventKey) *tcell.EventKey {
+	paths := c.GetTable().GetSelectedItems()
+	if len(paths) == 0 {
+		return nil
+	}
+
+	c.Stop()
+	defer c.Start()
+	c.showSuspendDialog(paths, true)
+
+	return nil
+}
+
+func (c *CronJob) setUnsuspend(evt *tcell.EventKey) *tcell.EventKey {
+	paths := c.GetTable().GetSelectedItems()
+	if len(paths) == 0 {
+		return nil
+	}
+
+	c.Stop()
+	defer c.Start()
+	c.showSuspendDialog(paths, false)
+
+	return nil
+}
+
+func (c *CronJob) showSuspendDialog(paths []string, suspend bool) {
+	title := "Suspend"
+	if !suspend {
+		title = "Unsuspend"
+	}
+	confirm := tview.NewModalForm(fmt.Sprintf("<%s>", title), c.makeSuspendForm(paths, suspend))
+	confirm.SetText(fmt.Sprintf("%s %s %d items", title, c.GVR(), len(paths)))
+	confirm.SetDoneFunc(func(int, string) {
+		c.dismissDialog()
+	})
+	c.App().Content.AddPage(suspendDialogKey, confirm, false, false)
+	c.App().Content.ShowPage(suspendDialogKey)
+}
+
+func (c *CronJob) makeSuspendForm(sel []string, suspend bool) *tview.Form {
+	f := c.makeStyledForm()
+	// suspended := strings.TrimSpace(c.GetTable().GetCell(c.GetTable().GetSelectedRowIndex(), c.GetTable().NameColIndex()+2).Text)
```

Comment:
Uncomment to read the active state

Reply:
Tnx, I was not sure how to access the current state! I solved with this

```
	suspended := strings.TrimSpace(c.GetTable().GetCell(c.GetTable().GetSelectedRowIndex(), c.GetTable().NameColIndex()+2).Text) == "true"

```

what do you think?